### DISCUSSION TITLE
Are users notified of organization role changes?

### DISCUSSION THREAD

**Discussion by wrosko** [community/community#29188] [Other Feature Feedback, Questions, & Ideas]
I'm wondering if users in an organization are notified if they hold `owner` privileges and their role is changed to `member` by another `owner`.

**Comment by Deadlyelder** [comment]
No, it does not provide any notification to the user about a change in the user's organization status. Furthermore, it does not inform if a user has been removed from the organization.

**Reply by dododoyo** [reply]
No it does send a notification when a user is removed from an organization
